pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -977,4 +977,12 @@ self: super: {
   # https://github.com/bos/configurator/issues/22
   configurator = dontCheck super.configurator;
 
+  # Use global libsass library in order to use hsass in the repl
+  hlibsass = overrideCabal super.hlibsass (drv: {
+    configureFlags = drv.configureFlags or [] ++ ["-fexternalLibsass"];
+    extraLibraries = drv.extraLibraries or [] ++ [ pkgs.libsass ];
```

Comment:
This is not the right way to do this. `hlibsass` expects an argument, `sass`, which is set to `null` because we don't have that identifier in Nixpkgs. Instead, we have `libsass`, so we should pass that one to the function instead:

    hlibsass = super.hlibsass.override { sass = pkgs.hlibsass };

Now, an even better solution that potentially benefits other packages as well is to teach the generator tool that `sass` exists and has a different name in our package set: https://github.com/NixOS/cabal2nix/commit/c0131db2907a2a42a2328e2c43dbbe43e5c9cc2f. That way, the expression is generated correctly in the first place.